src/android/StatusBar.java

```diff
@@ -118,194 +133,115 @@ else if(Build.VERSION.SDK_INT == Build.VERSION_CODES.KITKAT) {
      * @return                  True if the action was valid, false otherwise.
      */
     @Override
-    public boolean execute(final String action, final CordovaArgs args, final CallbackContext callbackContext) throws JSONException {
+    public boolean execute(final String action, final CordovaArgs args, final CallbackContext callbackContext) {
         LOG.v(TAG, "Executing action: " + action);
-        final Activity activity = this.cordova.getActivity();
-        final Window window = activity.getWindow();
 
-        if ("_ready".equals(action)) {
-            boolean statusBarVisible = (window.getAttributes().flags & WindowManager.LayoutParams.FLAG_FULLSCREEN) == 0;
-            callbackContext.sendPluginResult(new PluginResult(PluginResult.Status.OK, statusBarVisible));
-            return true;
-        }
+        switch (action) {
+            case ACTION_READY:
+                boolean statusBarVisible = (window.getAttributes().flags & WindowManager.LayoutParams.FLAG_FULLSCREEN) == 0;
+                callbackContext.sendPluginResult(new PluginResult(PluginResult.Status.OK, statusBarVisible));
+                return true;
 
-        if("isStatusBarOverlayingWebview".equals(action)) {
-            boolean statusBarVisible = (window.getAttributes().flags & WindowManager.LayoutParams.FLAG_FULLSCREEN) == 0;
-            callbackContext.sendPluginResult(new PluginResult(PluginResult.Status.OK, doOverlay && statusBarVisible));
-            return true;
-        }
```

Comment:
This seems outsystems specific, should not be removed

Reply:
nevermind, the code is still there, just missing the `isStatusBarOverlayingWebview` declaration